scala/src/main/org/apache/spark/api/csharp/CSharpBackendHandler.scala

```diff
@@ -176,31 +174,27 @@ class CSharpBackendHandler(server: CSharpBackend) extends SimpleChannelInboundHa
       }
     } catch {
       case e: Exception =>
-        // TODO - logError does not work now..fix //logError(s"$methodName on $objId failed", e)
         val jvmObj = JVMObjectTracker.get(objId)
         val jvmObjName = jvmObj match {
           case Some(jObj) => jObj.getClass.getName
           case None => "NullObject"
         }
-        // scalastyle:off println
-        println(s"[CSharpBackendHandler] $methodName on object of type $jvmObjName failed")
-        println(e.getMessage)
-        println(e.printStackTrace())
+        logError(s"On object of type $jvmObjName failed ${e.getMessage}")
+        e.printStackTrace()
```

Comment:
I follow the logic of original code and even in Spar source code `printStackTrace()` method is not contained in logger, such as: [Client.scala](https://github.com/apache/spark/blob/v2.0.0/core/src/main/scala/org/apache/spark/deploy/Client.scala#L142)